lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -699,6 +700,93 @@ FIRRTLType MemOp::getDataTypeOrNull() {
 // Statements
 //===----------------------------------------------------------------------===//
 
+static bool areConnectEquivalent(FIRRTLType destType, FIRRTLType srcType);
+
+/// Helper to implement the equivalence logic for a pair of bundle elements in a
+/// connect.
+static bool areElementsConnectEquivalent(
```

Comment:
Is this checking type equivalence or connect equivalence?  If the later, I'd suggest something like "areBundleElementEquivalent".  Per the comment below, please drop the tuple type, it is just obfuscating the code with `get<>`'s.